dev/App.js

```diff
@@ -3,9 +3,13 @@ import {hot} from 'react-hot-loader';
 import plotly from 'plotly.js/dist/plotly';
 import '../src/styles/main.scss';
 import 'react-select/dist/react-select.css';
-import ReactJson from 'react-json-view';
+import brace from 'brace'; // eslint-disable-line no-unused-vars
+import AceEditor from 'react-ace';
```

Comment:
can we do `import 'brace'` ?

we're loading those 2 resources down below:
import 'brace/mode/json';
import 'brace/theme/textmate';

that's not all we need?

Reply:
I'm not sure. I followed the readme from `react-ace`